gradle/maven.gradle

```diff
@@ -26,15 +26,13 @@ plugins.withId('maven') {
                         snapshotRepository(url: "https://oss.sonatype.org/content/repositories/snapshots/") {
                             authentication(userName: publishUsername, password: publishPassword)
                         }
-                    }
-                    else if (publishUrl == "jcenterSnapshot") {
+                    } else if (publishUrl == "jcenterSnapshot") {
```

Comment:
Would it become "dead" code at this point?  Meaning pushes to jcenter won't work anymore, is that correct?

Reply:
I think you're right. Good catch.